src/UrlProcessor.ts

```diff
@@ -35,16 +35,22 @@ export class UrlProcessor {
         }
 
         let normalUrl: string;
-        let result: string;
         if (this.isRelative(url)) {
+            // console.debug('UrlProcessor.process relative url: ' + url);
```

Comment:
There's a lot of commented out `console.debugs`, did you forget to remove them ?

Reply:
Nah, I can remove them, I just like keeping such for future debugging, but yah, that's just me.